### PR TITLE
Add finance grid sample

### PR DESCRIPTION
**Pull request by mddragnev**
To run this sample:

1. Navigate to the projects/FinanceGrid directory
2. npm install
3. npm run start

### CODE REVIEW

**Comment by sbayreva** [comment]
1. "filtering" should be with capital letter.
Screenshot 2024-12-04 at 10 12 38

2. The column should be called “Position”. The header and the chip should be right aligned
Screenshot 2024-12-04 at 10 41 27

3. The sorting icon should be on the left
Screenshot 2024-12-04 at 10 50 36

4. The whole column should be left aligned (header and chip). The sorting icon should be on the left.
Screenshot 2024-12-04 at 10 49 04

5.The sorting icons should be on the left
Screenshot 2024-12-04 at 10 54 32

6. There is some additional line between the pagination and the content of the grid.
Screenshot 2024-12-04 at 11 42 22

File: projects/FinanceGrid/src/app/app.config.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ApplicationConfig, provideZoneChangeDetection } from "@angular/core";
+import { provideRouter } from "@angular/router";
```
Comment (skrustev) [inline projects/FinanceGrid/src/app/app.config.ts:2]: A leftover import no longer in use I think

File: projects/FinanceGrid/src/app/finance-grid/finance-grid.component.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        <igx-avatar
+          [src]="getPathToImage(val)"
+          [shape]="'rounded'"
+        ></igx-avatar>
+        <span>{{ val }}</span>
+      </div>
+    </ng-template>
+  </igx-column>
+  <igx-column
+    field="positions"
```
Comment (skrustev) [inline projects/FinanceGrid/src/app/finance-grid/finance-grid.component.html:34]: Not really a big fan of each property being on a new line, but that's just me maybe ;D

File: projects/FinanceGrid/src/app/services/financial.service.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      newPrice,
+      dataRow.value.boughtPrice,
+      dataRow.positions
+    );
+    const newProfitLossPercentage = this.calculateProfitLossPercentage(
+      newProfitLossValue,
+      dataRow.value.boughtPrice,
+      dataRow.positions
+    );
+    const result = {
```
Comment (skrustev) [inline projects/FinanceGrid/src/app/services/financial.service.ts:98]: Just return the object, I guess? No need for a variable

**Comment by yradoeva** [comment]
Some more things related to design:

1. The scrollbars should be thicker, like these on the screenshot (from the [Ignite UI repo](https://github.com/IgniteUI/igniteui-angular)).
Pasted Graphic 34
In the demo app they look so:

Items per page


2. It would be good if the columns "Symbol," "Average Cost Per Share," and "Daily Change %" also have sorting functionality.

**Review by skrustev — APPROVED** [review]

**Pull request merged**